### ISSUE TITLE
Ctrl+H doesn't hide DevTools container

### ISSUE THREAD

**Issue by shanewilson** [gaearon/redux-devtools#54]
Hitting Ctrl+H hides the `LogMonitor` but not the `DebugPanel` - so you still see the box shadow and cannot click through.

https://github.com/gaearon/redux-devtools/blob/master/src/react/DebugPanel.js#L12
https://github.com/gaearon/redux-devtools/blob/master/src/react/LogMonitor.js#L115

Before Ctrl+H
```
<div style="position:fixed;z-index:10000;font-size:17px;overflow:hidden;opacity:1;color:white;right:0;top:0;bottom:0;max-height:100%;max-width:20%;min-width:300px;word-wrap:break-word;box-sizing:border-box;box-shadow:-2px 0 7px 0 rgba(0, 0, 0, 0.5);" data-reactid=".0.1">
<div ...>...stuff here...</div>
</div>
```

After Ctrl+H
```
<div style="position:fixed;z-index:10000;font-size:17px;overflow:hidden;opacity:1;color:white;right:0;top:0;bottom:0;max-height:100%;max-width:20%;min-width:300px;word-wrap:break-word;box-sizing:border-box;box-shadow:-2px 0 7px 0 rgba(0, 0, 0, 0.5);" data-reactid=".0.1">
<noscript data-reactid=".0.1.0"></noscript>
</div>
```

### Fix
Removing `min-width` seems to fix it

**Comment by nicinabox** [comment]
@gaearon what about just toggling the `DebugPanel` directly?

**Comment by gaearon** [comment]
@nicinabox 

It's currently a dumb component, it doesn't even know whether to show itself. I'd rather keep it this way.

**Comment by nicinabox** [comment]
Moving `min-width` to `LogMonitor` seems like a good solution then. Not sure about keeping `max-width` though. Seems to interfere with button visibility. Do you think the width will end up being configurable/draggable?

**Comment by shanewilson** [comment]
moving the `min-width` onto `LogMonitor` seemed like an easy fix - although I'm not sure why it needs a min width anyway.

**Comment by nicinabox** [comment]
It needs either `width` or `min-width` because the parent is `position: fixed;`

**Comment by gaearon** [comment]
Should be fixed in 1.0.2
